spec/unit/readerpaging_spec.lua

```diff
@@ -71,15 +71,15 @@ describe("Readerpaging module", function()
             UIManager:quit()
         end)
 
-        it("should scroll withtout crash backward on the first page", function()
+        it("should scroll without crash backward on the first page", function()
```

Comment:
If you're correcting these things anyway it should be something more like `should scroll backward on the first page without crashing` ;-)